internal/rbd/nodeserver.go

```diff
@@ -701,7 +701,16 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	if volumeID == "" {
 		return nil, status.Error(codes.InvalidArgument, "volume ID must be provided")
 	}
-	volumePath := req.GetVolumePath()
+
+	// Get volume path
+	// With Kubernetes version>=v19.0, expand request carries volume_path and
```

Comment:
version >= v1.19.0 

@agarwal-mudit ^^^

Reply:
Thanks for catching this